lib/control/v1/authenticate.js

```diff
@@ -18,6 +19,9 @@ function Authenticate(app, server, vtoken) {
   app.post('/v1/authenticate',
     (req, res, next) => {
       console.log('authenticate request recieved');
+      if (!global.Config.get('flag:unseal')){
```

Comment:
You don't need to to prepend the Config or Log getters with `global`. That's only for defining the variable in the global scope.